docs-compose/overwatch_docs/content/DataEngineer/Productionizing.md

```diff
@@ -55,6 +56,240 @@ created each run but it will be replaced on each subsequent run. If you would li
+
+After snapshot process is done the tables from Source ETL Database will be cloned in the target prefix location. The structure for target prefix location would be like below:
+
+![Snapshot_Target](/images/DataEngineer/snapshotTarget.png)
+
+All the tables data would be there in "targetPrefix/data" folder as shown below:
+
+![Snapshot_Target_Data](/images/DataEngineer/snapshotTarget_Data.png)
+
+If you look into the targetPrefix folder there would one another subdirectory called "clone_report". This folder contains the data regarding the report to show the status of your snapshot process for each table:
+
+![clone_report](/images/DataEngineer/clone_report.png)
+
+##### Snapshot through Databricks Job
+
+Snapshot process can also be configured using Databricks Job. For this we need to run SnapshotRunner Class.
+
+Below is the screenshot of the Job through which we run the snapshot process.
+
+1. **Type** - Jar
+2. **Main class** - com.databricks.labs.overwatch.pipeline.SnapshotRunner
+3. **Parameters for the Job** - Same as mentioned above for `Snapshot through Databricks Notebook`
+
+![Snapshot_Job](/images/DataEngineer/snapshot_job.png)
+
+
+#### Validation Functionality
+This is a special module in Snapshot Process. By default, it is part of Snapshot.process() but if user want to run it separately they can run it separately 
+to check whether the configuration they would provide for Snapshot process, are correct or not.
+
+Below is the code snippet for the validation function
+
+```scala
+import com.databricks.labs.overwatch.pipeline.Snapshot
+val sourceETLDB = "ow_bronze_snapshot_etl"
+val pipeline = "Bronze,Silver,Gold"
+val snapshotType = "Incremental"
+Snapshot.isValid(sourceETLDB,snapshotType,pipeline)
+```
+
+Upon running the above function you will get proper output whether the input parameters you would be using for Snapshot process,passed the validation check or not. One example of the same is as below:
+
+```
+ow_bronze_snapshot_etl is Overwatch Database and suitable for Snapshot
+Snapshot Type is Suitable for Snapshot Process. Provided SnapshotType value is Incremental
+Zone should be either Bronze,Silver or Gold. Provided Zone value is Bronze
+Zone should be either Bronze,Silver or Gold. Provided Zone value is Silver
+Zone should be either Bronze,Silver or Gold. Provided Zone value is Gold
+cloneLevel Type is Suitable for Snapshot Process. Provided cloneLevel value is DEEP
+Validation successful.You Can proceed with Snapshot process
+import com.databricks.labs.overwatch.pipeline.Snapshot
+res14: Boolean = true
+```
+
+### Restore Process
+Restore is the reverse process of Snapshot.In data world the term Restore is very common. It simply means if our current working space is corrupted then we can restore the current workspace from the snapshot location we have used earlier.
```

Comment:
Change to: "You can use the Restore process to bring back snapshotted data"

Reply:
Done